### DISCUSSION TITLE
Nice RF module DIO - requirements & limitations for APRS mode

### DISCUSSION THREAD

**Discussion by strud** [jgromes/RadioLib#574] [Q&A]
I am using a module from Nice RF : https://www.nicerf.com/products/detail/sx1268-wireless-module-lora1268f30.html

This is basically a 1268 chipset with power amplifier and RF switch etc.

NiceRF have only made the DIO1 & DIO3 available ie DIO2 is not broken out.

I have successfully operated this module with this library in LoRa mode, but I am now hoping to use it in the same project in APRS mode. 
The question I have is can it be made to work with the library as is? I note in the APRS example DIO2 is referenced, must this be used? 

I guess I am a little confused as to how the DIO are utilised by the library for particular modes.

**Comment by jgromes** [comment]
Unfortunately, SX126x cannot be used for APRS, as it cannot do AFSK. You *could* transmit APRS via 2-FSK, but at that point no APRS repeater/gateway is going to understand it.
